### PR TITLE
Some more twitter accounts

### PR DESCRIPTION
**Pull request by chrishenry**

### CODE REVIEW

**Comment by dwillis** [comment]
Thanks, @chrishenry. Some of these (justinamash, for example) are not official government twitter accounts but personal or campaign ones, which we don't include here.

**Comment by JoshData** [comment]
repjohnmica is a protected account.

**Comment by chrishenry** [comment]
I've removed `repjohnmica`.

**Comment by JoshData** [comment]
LacyClayMO1 and DanaRohrabacher also seem to be campaign accounts.

File: legislators-social-media.yaml [hunk 1]
```diff
@@ -1712,6 +1712,7 @@
     thomas: '01685'
     govtrack: 400279
   social:
+    twitter: repjeffmiller
```
Comment (konklone) [inline legislators-social-media.yaml]: Jeff Miller's [official website](http://jeffmiller.house.gov/) doesn't link to this account, but the Twitter account is verified and does link to his official website, so I think, it's :+1:.

**Comment by chrishenry** [comment]
Lacy Clay's account is quite active, and there doesn't seem to be any other account with day-to-day tweets. Despite the `Campaign` mention in the description, this seems like his official account.

Not sure how you determined DanaRohrabacher's account was focused on his campaign, a quick Find turns up no mention of it.

File: legislators-social-media.yaml [hunk 2]
```diff
@@ -3478,6 +3480,7 @@
     thomas: '01654'
     govtrack: 400074
   social:
+    twitter: lacyclaymo1
```
Comment (konklone) [inline legislators-social-media.yaml]: The [Twitter account](https://twitter.com/lacyclaymo1) says it's the "Official Campaign Twitter Account" and links to his campaign website, so this should not be included.

File: legislators-social-media.yaml [hunk 3]
```diff
@@ -5823,6 +5826,7 @@
     govtrack: 400343
     thomas: '00979'
   social:
+    twitter: DanaRohrabacher
```
Comment (konklone) [inline legislators-social-media.yaml]: His [official website](https://rohrabacher.house.gov/) doesn't link to this Twitter account, and the account links to a Facebook page. I would not add this, unless you called his legislative office directly to verify.

File: legislators-social-media.yaml [hunk 4]
```diff
@@ -3149,6 +3150,7 @@
     thomas: '01799'
     govtrack: 400648
   social:
+    twitter: repcharliedent
```
Comment (konklone) [inline legislators-social-media.yaml]: :+1: on this account.

**Comment by konklone** [comment]
> Lacy Clay's account is quite active, and there doesn't seem to be any other account with day-to-day tweets. Despite the Campaign mention in the description, this seems like his official account.

No, it's his campaign account, it links to the campaign website. He may just not have an official account. Sometimes that's the case.

File: legislators-social-media.yaml (hunk 2 above)
Comment (chrishenry) [inline legislators-social-media.yaml]: Have you guys read the content of both the Twitter feed and the website? They both go back years. Excluding it on a technicality seems a bit silly.

Comment (JoshData) [inline legislators-social-media.yaml]: @chrishenry Things that seem silly to you might be important to other users of this dataset.

Comment (dwillis) [inline legislators-social-media.yaml]: Agreed - the policy is to show official *government* accounts, not campaign or personal ones.

Comment (dwillis) [inline legislators-social-media.yaml]: I have, and I have no desire (nor the resources) to make individual decisions to include or exclude based on content. That's why we have a policy.

Comment (konklone) [inline legislators-social-media.yaml]: @chrishenry The dataset is designed to include accounts that are paid for with public funds and which represent official communications of their office. We rely on reasonable verification from the legislative office about the status of their accounts.

Offices are supposed to maintain strict separation of official funds and campaign funds, and official funds are not supposed to be used to further things like re-election efforts.

In practice, a campaign account may often look similar to an official account in terms of content, especially when expressing views on issues and legislations. However, there will be differences in what's appropriate for each account, and they will likely be maintained by different staff employed by different organizations.

The account is clear that it's a campaign account, and that's not what we wish to track here in this dataset about Congress' official membership and output.

Comment (konklone) [inline legislators-social-media.yaml]: I'll add that this dataset does frequently fall behind, and we have a fairly sophisticated script to harvest leads on new official accounts, mostly by scraping the HTML of official websites:

https://github.com/unitedstates/congress-legislators/blob/master/scripts/social_media.py

You would be welcome to run that script and use it to identify gaps that we should fill -- there almost certainly are some.

Comment (konklone) [inline legislators-social-media.yaml]: @chrishenry: Good idea, I just gave it a shot in #390.

**Pull request closed without merging**